### PR TITLE
feat(ingest): Auto Ingestion Data Platforms on GMS boot

### PR DESCRIPTION
**Pull request by chinmay-bhat**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestDataPlatformsStep.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+@Slf4j
+public class IngestDataPlatformsStep implements BootstrapStep {
+
+  private static final String PLATFORM_ASPECT_NAME = "dataPlatformInfo";
+
+  private final EntityService _entityService;
+
+  public IngestDataPlatformsStep(final EntityService entityService) {
```
Comment (dexter-mh-lee) [inline metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestDataPlatformsStep.java:38]: @RequiredArgsConstructor will take care of this

**Review by dexter-mh-lee — APPROVED** [review]
LGTM! Minor comment

File: metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestDataPlatformsStep.java (hunk 1 above)
Comment (chinmay-bhat) [reply metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestDataPlatformsStep.java:38]: Appartently not. It throws an error.
error: constructor IngestPoliciesStep in class IngestPoliciesStep cannot be applied to given types;
  final IngestPoliciesStep ingestPoliciesStep = new IngestPoliciesStep(_entityService);

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**